### DISCUSSION TITLE
Is it possible to install 2 quasar version?

### DISCUSSION THREAD

**Discussion by vitaly-agriocom** [quasarframework/quasar#12071] [General - Components / Directives / etc] [answered]
I would like to test quasar 2 / vue 3 keeping quasar 1 forold projects on my computer. Is it possible?

**Comment by MilosPaunovic** [comment]
Yes, it is possible.

**Comment by yusufkandemir** [comment, marked as answer]
Yes, it's possible.

All Quasar projects have their own local `quasar`, `@quasar/app`(_`quasar dev`, `quasar build`, etc._), etc. versions, which means you can simultaneously have different projects that have totally different major(v1, v2) and minor/patch(v1.1.0, v1.2.2, etc.) without changing or breaking anything.

For creating new projects with different versions:
The global CLI is called `@quasar/cli`, which is shared for all projects and used for global commands like `create`, `serve`, `info`, etc. For creating a new Quasar v2 project, you need to run `quasar create your-project-name`, and when creating v1 project, you need to run `quasar create --branch v1 your-project-name`.

**Reply by devpixde** [reply]
This does not work any more.
Tried `quasar create myproject`
Resulted in: 
`This command is now reserved only for scaffolding
 with CUSTOM starter kits or legacy v0.x projects.

 For scaffolding an official Quasar project please use this instead:

   yarn create quasar
   # or
   npm init quasar
